### PR TITLE
user registration and login changes

### PR DESCRIPTION
**Pull request by Ramlala-Yadav-Git**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "test": "nodemon src/index.js",
-    "start": "node src/index.js"
+    "start": "nodemon src/index.js"
```
Comment (Somesh-Kumar-Yadav) [inline package.json]: why nodemon?

File: src/index.js [hunk 2]
```diff
@@ -6,18 +6,22 @@ const connect = require("./config/mongoConfig");
...
 app.use(cors());
-app.use(urlencoded({ extended: false }));
+app.use(urlencoded({ extended: true }));
+app.use(express.urlencoded({extended: true }))
 app.use(json());
 app.use("*", cloudinaryConfig);
+app.use(multer().any())
 
-app.use("/book-my-hotel", router);
+app.use("/", router);
```
Comment (Somesh-Kumar-Yadav) [inline src/index.js]: do "/app"

File: package.json (hunk 1 above)
Comment (Ramlala-Yadav-Git) [reply package.json]: done

File: src/index.js (hunk 2 above)
Comment (Ramlala-Yadav-Git) [reply src/index.js]: done

**Pull request merged**